### ISSUE TITLE
Player won't show up

### ISSUE THREAD

**Issue by mbruhler** [remotion-dev/remotion#3128]
# Bug Report 🐛

Hi, something weird happened today while I was working on my project. After installing lottie-web and @remotion/lottie the Player dissappeared. Everything was working great until today. I also bumped the remotion packages versions and still nothing, Player is not there. What's interesting it was working, so I reverted everything to the tomorrow state and still nothing. There is no errors in console, even in the page source player is not there. I ran out of ideas what can be wrong.

I don't know what code should I provide as nothing throws any errors etc.

This is where Player is:
```
return (
        <div className="flex justify-center items-center w-screen ">
            <div className="flex justify-center gap-6 items-center w-full">
                <Player
                    ref={playerRef}
                    component={MyComposition}
                    durationInFrames={overallDuration}
                    fps={30}
                    compositionHeight={1920}
                    compositionWidth={1080}
                    controls
                    autoPlay
                    loop
                    className="shadow-2xl"
                    inputProps={{ scenes, backgroundMusic, logo: template?.logo || '' }}
                    style={{ width: '570px' }}
                />
                <div className="w-1/4">
                    <!-- Menu here -->
                </div>
            </div>
        </div>
    );
```

And this is a screenshot of the page:
![image](https://github.com/remotion-dev/remotion/assets/21124163/018f9659-0e93-4b0a-a466-7cdd5381dafe)

As you can see, no Player...

If I can provide anything more to help resolve this, I'm ready to do so :)

I'm running latest Chrome on Mac M2, it's a Vite project.

EDIT:
I tested on the completely fresh Vite project now and Player is also not visible. There are only 2 packages in this project `remotion` and `@remotion/player`

**Comment by alangabrielbs** [comment]
I'm having the same problem, I just created a project using [https://www.remotion.dev/templates/next](https://www.remotion.dev/templates/next) on my machine the Player doesn't appear, I deployed it to Vercel and the player doesn't appear either

**Comment by alangabrielbs** [comment]
I solved this problem by returning all remotion packages to version "4.0.54"

**Comment by JonnyBurger** [comment]
Thanks for reporting! This will be fixed in v4.0.64.

The cause was a Turborepo upgrade which somehow led to an old version of Remotion be re-published again.
